src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs

```diff
@@ -33,7 +33,8 @@ internal static partial class SymbolCompletionItem
             string filterText = null,
             SupportedPlatformData supportedPlatforms = null,
             ImmutableDictionary<string, string> properties = null,
-            ImmutableArray<string> tags = default)
+            ImmutableArray<string> tags = default,
+            bool isComplexTextEdit = false)
```

Comment:
for symbol completion, isn't it always false?  if you're going to touch symbol completion, i'd make this non-optional so callers have to think about this.

Reply:
oh, i see.  MemberInsertingProvider calls these.  hrmm... ok this can stay.